yarn.lock

```diff
@@ -763,24 +740,24 @@
   integrity sha512-BurYsht8MKdhvW2itUPPF8NkAhYtDdsCGHTSY7EzVvlmGP4jc9XrRZyfNwlt0zhB6MCMjHB1uNWwchtX7vBFjw==
 
 "@metamask/eth-json-rpc-provider@^2.1.0":
```

Comment:
Maybe we could make this a full upgrade since it appears to be a breaking change for the `params` properties?

```suggestion
"@metamask/eth-json-rpc-provider@^2.3.0":
```

Accompanying change in package.json:

```diff
diff --git a/package.json b/package.json
index b02d88c..0ed9698 100644
--- a/package.json
+++ b/package.json
@@ -25,7 +25,7 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
-    "@metamask/eth-json-rpc-provider": "^2.1.0",
+    "@metamask/eth-json-rpc-provider": "^2.3.0",
     "@metamask/safe-event-emitter": "^3.0.0",
     "@metamask/utils": "^8.1.0",
     "json-rpc-random-id": "^1.0.1",
```